### PR TITLE
Wither for 0.16 & 1.0

### PR DESCRIPTION
**Pull request by SuperXingKong**

### CODE REVIEW

File: src/slapper/Main.php [hunk 1]
```diff
@@ -176,6 +177,8 @@ public function onEnable() {
         Entity::registerEntity(SlapperStray::class, true);
         Entity::registerEntity(SlapperHusk::class, true);
         Entity::registerEntity(SlapperWitherSkeleton::class, true);
+        
+Entity::registerEntity(SlapperWither::class, true);
```
Comment (dktapps) [inline src/slapper/Main.php]: formatting?

Comment (SuperXingKong) [inline src/slapper/Main.php]: It can be used

Comment (robske110) [inline src/slapper/Main.php]: You should fix it anyways.

Comment (dktapps) [inline src/slapper/Main.php]: too late for that... he deleted his repo.

**Comment by SuperXingKong** [comment]
@dktapps I am sorry I can't understand English well.And now I create a new requests:)

**Pull request closed without merging**